pkg/corerp/frontend/controller/applications/graph_util_test.go

```diff
@@ -69,9 +70,7 @@ func Test_isResourceInEnvironment(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := isResourceInEnvironment(tt.args.ctx, tt.args.resource, tt.args.environmentName); got != tt.want {
-				t.Errorf("isResourceInEnvironment() = %v, want %v", got, tt.want)
-			}
+			require.Equal(t, tt.want, isResourceInEnvironment(tt.args.ctx, tt.args.resource, tt.args.environmentName))
```

Comment:
Thank you. Copilot generates this style of comparison, and I have to frequently point it out in PRs.